documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1030,6 +1030,48 @@ You cannot use the `hybrid` mining strategy if the xref:oracle-property-lob-enab
+=== Using a downstream mining database
+
+You can use the {prodname} Oracle connector to capture changes from an Oracle Database mining database.
+Log mining operations consume system resources and increase the load on a database, which can result in decreased performance.
+Configuring the connector to perform change data capture processing on a secondary mining database, can reduce the load on the primary database.
+
+[NOTE]
+====
+This feature is in an incubating state. This means that exact semantics, configuration options, and other details may change in future revisions based on feedback.
+Please let us know your specific requirements or if you encounter any problems while using this feature.
+====
+
+.Prerequisites
+
+* An Oracle downstream mining database is configured and synchronized with the primary instance.
+* The downstream mining database has access to the external dictionary file that the primary database instance generates.
+* The connector configuration includes following key properties:
+`log.mining.strategy`:: This property is set to `dictionary_from_file`.
+`internal.log.mining.read.only`:: This property is set to `true` to enable capturing changes from a downstream mining database, rather than from the primary instance.
+`log.mining.read.only.hostname`:: Specifies the hostname of the downstream mining database from which you want the {prodname} Oracle connector to read and capture transaction logs.
+Because the connector uses this downstream mining instance to capture changes, changes must be replicated from the primary to the downstream instance before {prodname} can emit change events.
+`log.mining.path.dictionary`:: Specifies the absolute file path on the downstream mining database server where Oracle LogMiner can access the data dictionary file that is exported from the primary Oracle instance.
+The LogMiner process on the downstream database must have access to the specified path.
+
+==== How this works
+
+When {prodname} Oracle connector is configured to capture changes from a downstream mining database, the connector operates as follows:
+
+1. The primary Oracle instance generates an external dictionary file containing the database schema and metadata information.
+2. This dictionary file is made available to the downstream mining database at the specified path.
+3. The connector connects to the downstream mining database and uses LogMiner with the external dictionary file to read transaction logs.
+4. Changes are captured from the downstream mining database without impacting the primary database performance.
+
+==== Important considerations
+
+1. The dictionary file must be regularly updated on the primary instance to ensure schema changes are properly reflected.
+2. Ensure that network and file access settings permit the downstream mining database to access the dictionary file and redo logs on the primary database.
+3. Monitor the dictionary file synchronization to prevent issues with schema change detection.
```

Comment:
```suggestion
* Ensure that network and file access settings permit the mining database to access the dictionary file and redo logs on the primary database.
* Keep the dictionary file on the primary instance updated to ensure that it properly reflects schema changes. 
* To prevent issues with schema change detection, keep the dictionary file on the mining database synchronized with the file on the primary.
```
WRT the synchronization item, I'm assuming the intent is to maintain equivalent states in the primary and downstream versions of the dictionary.